.circleci/config.yml

```diff
@@ -86,6 +87,127 @@ jobs:
+      - restore_cache:
+          keys:
+            - v1-gt_cache_54ranks-<<parameters.backend>>-{{ checksum "gt4py_version.txt" }}
+            - v2-savepoints-c12_54ranks_standard-{{ checksum "Makefile.data_download" }}
+            - v2-pip_cache-{{ checksum "requirements_dev.txt" }}-{{ checksum "constraints.txt" }}
+      - run:
+          name: set up environment
+          command: |
+            pip3 install --upgrade setuptools wheel
+            pip3 install -r requirements_dev.txt -c constraints.txt
+      - save_cache:
+          key: v2-pip_cache-{{ checksum "requirements_dev.txt" }}-{{ checksum "constraints.txt" }}
+          paths:
+            - /home/circleci/.cache/pip
+      - run:
+          name: run tests
+          command: |
+            TEST_ARGS="--backend=<<parameters.backend>> -v -s" CONTAINER_CMD="" NUM_RANKS=54 EXPERIMENT="c12_54ranks_standard" FV3_PATH=$(pwd)/fv3core make savepoint_tests_mpi
+          no_output_timeout: 3h
+      - save_cache:
+          key: v1-gt_cache_54ranks-<<parameters.backend>>-{{ checksum "gt4py_version.txt" }}
+          paths:
+            - .gt_cache
+            - .gt_cache_000000
+            - .gt_cache_000001
+            - .gt_cache_000002
+            - .gt_cache_000003
+            - .gt_cache_000004
+            - .gt_cache_000005
+            - .gt_cache_000006
+            - .gt_cache_000007
+            - .gt_cache_000008
+            - .gt_cache_000009
+            - .gt_cache_000010
+            - .gt_cache_000011
+            - .gt_cache_000012
+            - .gt_cache_000013
+            - .gt_cache_000014
+            - .gt_cache_000015
+            - .gt_cache_000016
+            - .gt_cache_000017
+            - .gt_cache_000018
+            - .gt_cache_000019
+            - .gt_cache_000020
+            - .gt_cache_000021
+            - .gt_cache_000022
+            - .gt_cache_000023
+            - .gt_cache_000024
+            - .gt_cache_000025
+            - .gt_cache_000026
+            - .gt_cache_000027
+            - .gt_cache_000028
+            - .gt_cache_000029
+            - .gt_cache_000030
+            - .gt_cache_000031
+            - .gt_cache_000032
+            - .gt_cache_000033
+            - .gt_cache_000034
+            - .gt_cache_000035
+            - .gt_cache_000036
+            - .gt_cache_000037
+            - .gt_cache_000038
+            - .gt_cache_000039
+            - .gt_cache_000040
+            - .gt_cache_000041
+            - .gt_cache_000042
+            - .gt_cache_000043
+            - .gt_cache_000044
+            - .gt_cache_000045
+            - .gt_cache_000046
+            - .gt_cache_000047
+            - .gt_cache_000048
+            - .gt_cache_000049
+            - .gt_cache_000050
+            - .gt_cache_000051
+            - .gt_cache_000052
+            - .gt_cache_000053
+            - .gt_cache_000054
+            - .gt_cache_000055
```

Comment:
We can only avoid this long list if we can put these caches in a common subdirectory, e.g. `.gt_cache/<rank>/`.

Reply:
Let's do that! If you set the env var `GT_CACHE_DIR_ROOT` it will customize the location

https://github.com/GridTools/gt4py/blob/master/src/gt4py/config.py#L66.